Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gradle:jdk11
+
+RUN apt update || true \
+    && apt install jq -y
+ENV SLEEP_DURATION 5s
+COPY build.sh .
+COPY BP-BASE-SHELL-STEPS/functions.sh .
+ENV ACTIVITY_SUB_TASK_CODE GRADLE_EXECUTE
+
```

Comment:
We should have a default INSTRUCTIOn as well i.e in maven mvn compile is default arg